hybrid/compartment_examples/inter_comp_call/secure-redirect_clr/Makefile.morello-hybrid

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 The CapableVMs "CHERI Examples" Contributors.
```

Comment:
Not 2022?

Reply:
Updated in https://github.com/capablevms/cheri-examples/pull/57/commits/e20156f8ebe99ccfd7a7bd113ff907bdc5d2ac92.